README.md

```diff
@@ -3,77 +3,86 @@
 [![codecov](https://codecov.io/gh/conda-incubator/conda-project/branch/main/graph/badge.svg?token=XNRS8JKT75)](https://codecov.io/gh/conda-incubator/conda-project)
 [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/conda-incubator/conda-project/main.svg)](https://results.pre-commit.ci/latest/github/conda-incubator/conda-project/main)
 
-Tool for encapsulating, running, and reproducing projects with conda environments
+
+Tool for encapsulating, running, and reproducing projects with conda environments.
 
 This package is intended as a successor to [Anaconda Project](https://github.com/Anaconda-Platform/anaconda-project).
-Please continue to use Anaconda Project until it has been declared deprecated and conda-project has
-stabilized.
 
-## Setup for development
+## Why?
+
+Sharing your work is more than sharing the script file or notebook with your code. To help your fellow users it is
```

Comment:
```suggestion
Sharing your work is more than sharing your code in a script file or notebook. To make your work properly reproducible, it is necessary to include the list of required third-party dependencies, specifications for how to run your code, and
any other files that it may need.
```